### ISSUE TITLE
Functions in fft_utils.py does not match version of torch>=1.8

### ISSUE THREAD

**Issue by yangpuPKU** [facebookresearch/active-mri-acquisition#55]
Thanks for this repository. 

In the file [activemri/experimental/cvpr19_models/models/fft_utils.py](https://github.com/facebookresearch/active-mri-acquisition/blob/main/activemri/experimental/cvpr19_models/models/fft_utils.py), the functions of torch.fft, torch.ifft and torch.rfft of torch<=1.7 are used for the implementation of the Fast Fourier Transform. However, these functions are dropped in the new version of torch>=1.8 and replaced with torch.fft.fft, torch.fft.ifft and torch.fft.rfft. What's more, the packages this code depends on include [fastmri](https://github.com/facebookresearch/fastMRI) which depends on torch>=1.8. The above problems will lead to a bug when running the [intro notebook](https://github.com/facebookresearch/active-mri-acquisition/blob/master/docs/notebooks/miccai_example.ipynb)

It would be nice to update fft_utils.py to the new version. 

Thanks, 
Pu Yang

**Comment by tianweiy** [comment]
you are right. In my own codebase, I wrote an if else statement to check which version of fft / ifft we should use. I will update this soon
